apps/meteor/client/startup/actionButtons/starMessage.ts

```diff
@@ -19,7 +20,8 @@ Meteor.startup(() => {
 			const { message = messageArgs(this).msg } = props;
 
 			try {
-				await sdk.call('starMessage', { ...message, starred: true });
+				await sdk.rest.post('/v1/chat.starMessage', { messageId: message._id });
+				starMessage(message, true);
```

Comment:
What will happen if `starMessage()` fails and throws and error? Shouldn't `['rooms', message.rid, 'starred-messages']` queries be invalidated regardless of this?

Reply:
Maybe we should put this query invalidation in a `finally` block, since it should happen regardless of try or catch block.